tutorial/markdown/java/spring-boot/spring-boot.md

```diff
@@ -0,0 +1,517 @@
+  - [Model](#model)
+  - [Controller](#controller)
+  - [Service](#service)
+  - [Repository](#repository)
+- [Mapping Workflow](#mapping-workflow)
+  - [GET Mapping Workflow](#get-mapping-workflow)
+  - [POST Mapping Workflow](#post-mapping-workflow)
+  - [PUT Mapping Workflow](#put-mapping-workflow)
+  - [DELETE Mapping Workflow](#delete-mapping-workflow)
+- [Custom SQL++ Queries](#custom-sql-queries)
+  - [Get all airlines by country](#get-all-airlines-by-country)
+  - [Get all airlines by destination airport](#get-all-airlines-by-destination-airport)
+- [Running The Tests](#running-the-tests)
+  - [Run Individual Tests](#run-individual-tests)
+- [Project Setup Notes](#project-setup-notes)
+- [Contributing](#contributing)
+- [Appendix](#appendix)
+  - [Extending API by Adding New Entity](#extending-api-by-adding-new-entity)
+  - [Running Self Managed Couchbase Cluster](#running-self-managed-couchbase-cluster)
+  - [Swagger Documentation](#swagger-documentation)
+
+## Getting Started
+
+### Prerequisites
+
+To run this prebuilt project, you will need:
+
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/dotnet-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+  - To run this tutorial using a self managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- Java SDK 17 or higher installed
+- Code Editor installed (IntelliJ IDEA, Eclipse, or Visual Studio Code)
+- Maven command line
+
+### Source Code
+
+```shell
+git clone https://github.com/couchbase-examples/java-springboot-quickstart.git
+```
+
+### Install Dependencies
+
+```shell
+mvn package
+```
+
+> Note: Maven packages auto restore when building the project in IntelliJ IDEA or Eclipse depending on IDE configuration.
+
+### Database Server Configuration
+
+- The `CouchbaseConfig` class is a Spring configuration class responsible for setting up the connection to a Couchbase database in a Spring Boot application. It defines two beans:
+
+  - `getCouchbaseCluster()`: This bean creates and configures a connection to the Couchbase cluster using the provided hostname, username, and password.
+
+  - `getCouchbaseBucket(Cluster cluster)`: This bean creates a Couchbase bucket within the cluster if it doesn't already exist and returns the Bucket object associated with the specified bucket name.
+
+### Application Properties
+
+You need to configure the connection details to your Couchbase Server in the application.properties file located in the src/main/resources directory.
+
+In the connection string, replace `DB_CONN_STR` with the connection string of your Couchbase cluster. Replace `DB_USERNAME` and `DB_PASSWORD` with the username and password of a Couchbase user with access to the bucket.
+
+The connection string should be in the following format:
+
+```properties
+spring.couchbase.bootstrap-hosts=couchbases://cb.fwu-oviwcpjq4v.cloud.couchbase.com
```

Comment:
Please remove the valid Capella connection string from here

Reply:
removed